src/Atmos/Model/radiation.jl

```diff
@@ -14,20 +14,20 @@ function preodefun!(::RadiationModel, aux::Vars, state::Vars, t::Real)
 end
 function integrate_aux!(::RadiationModel, integ::Vars, state::Vars, aux::Vars)
 end
-function flux_radiation!(::RadiationModel, flux::Grad, state::Vars,
+function flux_radiation!(::RadiationModel, m::AtmosModel, flux::Grad, state::Vars,
                          aux::Vars, t::Real)
 end
 
 struct NoRadiation <: RadiationModel
 end
 
 """
-  StevensRadiation <: RadiationModel
+  DYCOMSRadiation <: RadiationModel
 
-Stevens et. al (2005) version of the δ-four stream model used to represent radiative transfer. 
+Stevens et. al (2005) approximation of longwave radiative fluxes in DYCOMS.
 Analytical description as a function of the liquid water path and inversion height zᵢ
 """
-struct StevensRadiation{FT} <: RadiationModel
+struct DYCOMSRadiation{FT} <: RadiationModel
   "κ [m^2/s] "
```

Comment:
The units here are wrong. `\kappa` should be a mass absorption coefficient [m^2/kg].